dataikuapi/fm/instances.py

```diff
@@ -406,6 +411,14 @@ class FMNodeType(Enum):
     AUTOMATION = "automation"
     GOVERN = "govern"
 
+# Python2 emulated enum. to be removed on Python2 support removal
+def get_from_string(s):
```

Comment:
should we define this method as a `@staticmethod` of class `FMNodeType`?
here, it is at the same level as the FM classes related to DSS instances in this file

Reply:
indeed, it would be nicer to hide this method